### ISSUE TITLE
Exploring options where Chromebooks and Chromebase  used in an organisation.  Dont want to pin grabngo app to all users.

### ISSUE THREAD

**Issue by ericwhiteau** [google/loaner#107]
### Expected Behaviour
Organisations that have embraced Chromebooks do not want to load grabngo  app on Chromebooks and Chromebases that are not part of the Loan program.

They would like the grabngo app to load only on the loaner devices.

I understand that the app is pinned for all users in the Org, which is how it shows as a popup app when the loan device is first logged in.

Has anyone found a way to achieve this without pinning it to the all users in Org?

Thanks in advance for advice / comments.

cheers.

**Comment by templesnr** [comment]
Hi @Eric - we don't pin the app, it's force - installed though. I did ask
about this when we did our first build but I was told it couldn't be done.
Maybe someone has found a way around it.
Temple

**Comment by ericwhiteau** [comment]
Thanks yes, I tried just force install, (without pin)  which works in a way but not the same User Experience.
Users have to know to launch the app, and it seems to just allocate the default loan period automatically.

cheers.

**Comment by templesnr** [comment]
Yes - I have an email template to tell them how to find the app and pin
it!  I also tell them about the web app too.  Maybe that is a possibility -
to push a bookmark with the web app url to all users.
Temple
